include/aspect/material_model/reactive_fluid_transport.h

```diff
@@ -137,6 +141,10 @@ namespace aspect
 
         // Time scale for fluid release and absorption.
         double fluid_reaction_time_scale;
+        double max_peridotite_water;
+        double max_gabbro_water;
```

Comment:
Look farther ahead, it looks like this and the other variables below are currently not being used to enforce upper limits? Is the plan to add them in a forthcoming commit?

Reply:
These are used in the function `tian_equilibrium_bound_water_content()` to limit the bound water for the respective rock type.